### ISSUE TITLE
CommonJS require paths when proto files are in subdirectories

### ISSUE THREAD

**Issue by rwlincoln** [grpc/grpc-web#244]
Lets say we have `.proto` files organised into directories:
```
apis/myproject/myapi/v1/myapi.proto
apis/myproject/otherapi/v1/otherapi.proto
```
Generating code involves including the `apis` directory:
```
$ protoc -I./apis  ...  ./apis/myproject/myapi/v1/myapi.proto
```
The generated code might be output to:
```
src/myproject/myapi/v1/
src/myproject/otherapi/v1/
```
The `google/protobuf` and `grpc/grpc` plug-ins generated `require` calls like this:
```
... = require("../../../myproject/myapi/v1/myapi_pb.js");
```
However, the `grpc-web` plug-in generates `require` calls like this:
```
... = require('./myproject/myapi/v1/myapi_pb.js');
```
The generated `require` paths should account for the directory structure relative to the import path.

**Comment by rwlincoln** [comment]
That is correct. I would like the generated `grpc-web` code to be in the same directory as all the other generated JS code (Protobuf, gRPC, TypeScript definitions) for that package. For example:

```
src/myproject/myapi/v1/myapi_pb.js
src/myproject/myapi/v1/myapi_pb.d.ts
src/myproject/myapi/v1/myapi_grpc_pb.js
src/myproject/myapi/v1/myapi_grpcweb_pb.js
```

As you point out, the `grpc-web` code can be placed in the top-level `src` directory and the `require` path is valid. However, when dealing with several APIs, with the goal of publishing the source and distributing packages, I believe the best practice is to have all the code organised into a hierarchy of directories. The `google/protobuf`, `grpc/grpc` and `improbable-eng/ts-protoc-gen` plug-ins create the `myproject/myapi/v1/` directory structure when passed `:./src`.

**Comment by rwlincoln** [comment]
As in https://github.com/grpc/grpc-web/issues/245#issuecomment-412767686, in order to maintain the current behaviour, the `out=` configuration option could be made optional and when omitted the plug-in could be made to work in a way that is consistent with the `google/protobuf` and `grpc/grpc` plug-ins.

**Comment by rogchap** [comment]
@rwlincoln @stanley-cheung 
I'm still seeing this behaviour, but only for the typescript output

The generated typescript tries to import from `"./myproject/myapi/v1/myapi_pb"` and not `"../../../myproject/myapi/v1/myapi_pb"`

I think the same fix in #249 needs to be applied to the typescript output.

**Comment by rogchap** [comment]
This seems to have been fixed by #360
